pkg/controller/migmigration/dvm.go

```diff
@@ -8,6 +8,7 @@ import (
 	dvmc "github.com/konveyor/mig-controller/pkg/controller/directvolumemigration"
```

Comment:
FYI @alaypatel07 typically we have put shared consts like `dvmc.MigrationFailed` in `migapi` (the package we alias as migapi) so that we don't end up with circular references (which will break the build) if two controllers need to interact with each other in some way.